client/my-sites/customer-home/controller.jsx

```diff
@@ -87,26 +78,12 @@ export async function maybeRedirect( context, next ) {
 		// We need to make sure that sites on the eCommerce plan actually have WooCommerce installed before we redirect to the WooCommerce Home
 		// So we need to trigger a fetch of site plugins
 		fetchPromise.then( () => {
-			const refetchedState = context.store.getState();
-			const siteUrl = getSiteUrl( refetchedState, siteId );
-
+			const siteUrl = getSiteUrl( state, siteId );
 			if ( siteUrl !== null ) {
-				const currentPlan = getCurrentPlan( refetchedState, siteId );
-				const purchase = getByPurchaseId( refetchedState, currentPlan?.id );
-
-				const isWooExpressPlatform =
-					isSiteOnWooExpress( refetchedState, siteId ) || purchase?.isWooExpressTrial;
-				const shouldUseCalypsoMyHome = ! isWooExpressPlatform;
-
+				const refetchedState = context.store.getState();
 				const installedWooCommercePlugin = getPluginOnSite( refetchedState, siteId, 'woocommerce' );
 				const isSSOEnabled = !! isJetpackModuleActive( refetchedState, siteId, 'sso' );
-
-				if (
-					isSSOEnabled &&
-					installedWooCommercePlugin &&
-					installedWooCommercePlugin.active &&
-					! shouldUseCalypsoMyHome
-				) {
+				if ( isSSOEnabled && installedWooCommercePlugin && installedWooCommercePlugin.active ) {
```

Comment:
I know it is something that you did not introduce, you are just reverting. Interesting condition though. Makes me wonder whether the redirection is applied to Creator plan users when they have Woo Commerce plugin installed and active. Just an observation. 🙂

Reply:
Interesting. It actually might happen. We should test this tomorrow.

Reply:
If that is the case, this happen anyway, even if we don't revert, right?
Because the `isSSOEnabled && installedWooCommercePlugin && installedWooCommercePlugin.active` conditions already exists today.

Reply:
Yup, it's not a blocker, if its an issue, it already existed, and if it needs to be fixed, tackle it separately.

Reply:
> Interesting. It actually might happen. We should test this tomorrow.

Just noting that sites with the Creator plan are not affected: there's a `isSiteOnWooExpressEcommerceTrial( state, siteId )` check higher in the file which prevents the redirection to WP Admin from happening.

This function would also benefit from name update - as the name suggests it is checking for Trial only - but it checks for non-trial plans as well. → Will add a new bullet point to our clean-up task (https://github.com/Automattic/dotcom-forge/issues/7194).